### PR TITLE
Unity server for : https://github.com/zkzhao333/play-unity-plugins

### PR DESCRIPTION
**Pull request by RafaelHiguera**

### CODE REVIEW

File: UnityServer/functions/tslint.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{
```
Comment (Qing451800) [inline UnityServer/functions/tslint.json:1]: JSON doesn't support comment.

File: UnityServer/functions/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (Qing451800) [inline UnityServer/functions/tsconfig.json:1]: Do you need to check in this?

File: ClassyTaxiServer/.firebase/hosting.d2ViYXBw.cache [hunk 3]
```diff
@@ -0,0 +1,8 @@
+README.md,1595281223409,cd85bc4a1ba07022f391a0e41d48b32523afade624d8c8327c482d3deb817e87
```
Comment (Qing451800) [inline ClassyTaxiServer/.firebase/hosting.d2ViYXBw.cache:1]: Do not check in this file

File: UnityServer/.firebaserc [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Qing451800) [inline UnityServer/.firebaserc:1]: Do not check in this file

File: UnityServer/.gitignore [hunk 5]
```diff
@@ -0,0 +1,65 @@
+# Logs
```
Comment (Qing451800) [inline UnityServer/.gitignore:1]: Do not check in this file

File: UnityServer/firebase.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Qing451800) [inline UnityServer/firebase.json:1]: Do you need this file?

File: UnityServer/functions/.gitignore [hunk 7]
```diff
@@ -0,0 +1,8 @@
+## Compiled JavaScript files
```
Comment (Qing451800) [inline UnityServer/functions/.gitignore:1]: Do not check in this file

File: UnityServer/functions/lib/keys/dummyKey.json [hunk 8]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (Qing451800) [inline UnityServer/functions/lib/keys/dummyKey.json:1]: Do not check in this file

File: ClassyTaxiServer/.firebase/hosting.d2ViYXBw.cache (hunk 3 above)
Comment (Qing451800) [reply ClassyTaxiServer/.firebase/hosting.d2ViYXBw.cache:1]: And remove other .DS_Store files

Comment (RafaelHiguera) [reply ClassyTaxiServer/.firebase/hosting.d2ViYXBw.cache:1]: Yep, will do

File: UnityServer/.firebaserc (hunk 4 above)
Comment (RafaelHiguera) [reply UnityServer/.firebaserc:1]: Yep

File: UnityServer/firebase.json (hunk 6 above)
Comment (RafaelHiguera) [reply UnityServer/firebase.json:1]: Yes, that will be the settings for setting up firebase in the server

File: UnityServer/.gitignore (hunk 5 above)
Comment (RafaelHiguera) [reply UnityServer/.gitignore:1]: Will remove it

File: UnityServer/firebase.json (hunk 6 above)
Comment (RafaelHiguera) [reply UnityServer/firebase.json:1]: Yep, is the config file for firebase

File: UnityServer/functions/lib/keys/dummyKey.json (hunk 8 above)
Comment (RafaelHiguera) [reply UnityServer/functions/lib/keys/dummyKey.json:1]: Yep, forgot to delete it

File: UnityServer/functions/tsconfig.json (hunk 2 above)
Comment (RafaelHiguera) [reply UnityServer/functions/tsconfig.json:1]: Yep, is part of the server config

File: UnityServer/functions/.gitignore (hunk 7 above)
Comment (RafaelHiguera) [reply UnityServer/functions/.gitignore:1]: 👍

**Pull request closed without merging**